tests/test_csc.py

```diff
@@ -151,6 +156,64 @@ def setUp(self):
             self.assertEqual(alarm.maxSeverity, AlarmSeverity.NONE)
             self.assertFalse(alarm.acknowledged)
 
+    async def test_mute(self):
+        """Test the mute and unmute command."""
+        async with Harness(config_dir=TEST_CONFIG_DIR) as harness:
+            await salobj.set_summary_state(harness.remote, state=salobj.State.ENABLED,
+                                           settingsToApply="enabled.yaml")
+            nrules = len(harness.csc.model.rules)
+
+            user1 = "test_mute 1"
+            # mute all alarms for a short time,
+            # then wait for them to unmute themselves
+            await harness.remote.cmd_mute.set_start(name="Enabled.*",
+                                                    duration=0.1,
+                                                    severity=AlarmSeverity.SERIOUS,
+                                                    mutedBy=user1,
+                                                    timeout=STD_TIMEOUT)
+            muted_names = []
+            while len(muted_names) < nrules:
+                data = await harness.remote.evt_alarm.next(flush=False, timeout=STD_TIMEOUT)
+                self.assertEqual(data.mutedSeverity, AlarmSeverity.SERIOUS)
+                self.assertEqual(data.mutedBy, user1)
+                muted_names.append(data.name)
+            self.assertEqual(set(muted_names), set(harness.csc.model.rules))
+
+            unmuted_names = []
+            while len(unmuted_names) < nrules:
+                data = await harness.remote.evt_alarm.next(flush=False, timeout=STD_TIMEOUT)
+                self.assertEqual(data.mutedSeverity, AlarmSeverity.NONE)
+                self.assertEqual(data.mutedBy, "")
+                unmuted_names.append(data.name)
+            self.assertEqual(set(unmuted_names), set(harness.csc.model.rules))
+
+            # now mute one rule for a long time, then explicitly unmute it
+            user2 = "test_mute 2"
+            full_name = "Enabled.ScriptQueue:2"
+            self.assertIn(full_name, harness.csc.model.rules)
+            await harness.remote.cmd_mute.set_start(name=full_name,
+                                                    duration=5,
+                                                    severity=AlarmSeverity.SERIOUS,
+                                                    mutedBy=user2,
+                                                    timeout=STD_TIMEOUT)
+            data = await harness.remote.evt_alarm.next(flush=False, timeout=STD_TIMEOUT)
+            self.assertEqual(data.name, full_name)
+            self.assertEqual(data.mutedSeverity, AlarmSeverity.SERIOUS)
+            self.assertEqual(data.mutedBy, user2)
+            # that should be the only alarm event from mute
```

Comment:
"from mute"?

Reply:
How about (and lengthening the timeout):
```
            # There should be the only alarm event from the mute command.
            with self.assertRaises(asyncio.TimeoutError):
                await harness.remote.evt_alarm.next(flush=False, timeout=1)
```
and similarly for the unmute test shortly thereafter